src/Microsoft.AspNetCore.Server.IntegrationTesting/breakingchanges.netcore.json

```diff
@@ -0,0 +1,78 @@
+[
+    {
+      "TypeId": "public abstract class Microsoft.AspNetCore.Server.Testing.ApplicationDeployer : Microsoft.AspNetCore.Server.Testing.IApplicationDeployer",
```

Comment:
The base lines were for 1.1.0. The timeline is as follows:
1. Assembly changed from M.A.S.Testing to M.A.S.IntegrationTesting in 1.1.0-preview1 (https://github.com/aspnet/Hosting/commit/f8189a2296e56b6b497afb98d49b004e283a93e5)
2. Baselines were created after the assemblies are renamed (https://github.com/aspnet/Hosting/commit/911da31476615a5ce4bfe2c9a59cffeddb8c2422)

I don't think we delete the baselines since it was created with the current assembly names. We should just update the breakingchanges.netcore.json with the removals.